crates/biome_package/src/node_js_package/package_json.rs

```diff
@@ -186,9 +382,10 @@ impl DeserializationVisitor for PackageJsonVisitor {
                 "type" => {
                     result.r#type = Deserializable::deserialize(ctx, &value, &key_text);
                 }
-                _ => {
-                    // each package can add their own field, so we should ignore any extraneous key
-                    // and only deserialize the ones that Biome deems important
+                key => {
+                    if let Some(value) = value.into_json_value() {
+                        result.raw_json.insert(key.into(), value.into());
```

Comment:
Why do we need raw JSON here?

Reply:
It’s because `oxc-resolver` wants us to implement these getter functions that can pull fields out of the `package.json` using arbitrary paths.

I considered just limiting our support to a fixed set of paths, so we can pull it from pre-parsed static structures, but I tried a similar thing in the `oxc-resolver` repo when I introduced the traits, and it made the tests fail. I haven’t gone too deep into figuring out why, but I figured I just play it safe :)